### PR TITLE
feat: add landing page

### LINKED ISSUE #19: Add landing page

### PR DESCRIPTION
**Pull request by fmaclen**
Closes #19 

- adds landing page (not final)
- adds `PUBLIC_IS_DEMO` flag
- adds tests for _demo_ environment
- implements TailwindCSS with default settings
- Partially satisfies #11

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,9 @@
 		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
 		"format": "prettier --write .",
 		"lint": "prettier --check . && eslint .",
-		"test:e2e": "playwright test",
-		"test": "npm run test:e2e"
+		"test:e2e": "playwright test --grep-invert @demo",
+		"test:e2e:demo": "playwright test --config=playwright.config.demo.ts --grep @demo",
+		"test": "npm run test:e2e && npm run test:e2e:demo"
```
Comment (fmaclen) [inline package.json:15]: We now have 2 test runs, one for tests with a `tag: '@demo'` and the default one for all tests that aren't tagged.

File: playwright.config.demo.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+export default {
+	...baseConfig,
+	webServer: {
+		...baseConfig.webServer,
+		env: {
+			PUBLIC_IS_DEMO: 'true'
+		}
+	}
+};
```
Comment (fmaclen) [inline playwright.config.demo.ts:11]: We are passing the `PUBLIC_IS_DEMO: 'true'` environment variable so the app get's re-built with that flag. This allows us to dynamically render the landing page _around_ the app functionality. Eventually this flag will allow us to turn on/off certain features we want to keep for the full-version releases.

File: src/routes/+layout.svelte [hunk 3]
```diff
@@ -15,10 +17,66 @@
...
+	}
+
+	.footer {
+		@apply container mx-auto flex flex-row items-center justify-center gap-4;
+
+		nav {
+			@apply ml-auto flex flex-row items-center justify-center gap-4;
+		}
+	}
+</style>
```
Comment (fmaclen) [inline src/routes/+layout.svelte:82]: These styles are WIP.

**Review by GregoMac1 — APPROVED** [review]
This looks good! I don't know if you're going to keep implementing the styles (I mean because you marked them as a WIP), but we could also leave them for #13.

**Comment by fmaclen** [comment]
> I don't know if you're going to keep implementing the styles

@GregoMac1 not sure yet, it depends on how are we doing with the time we have left. I'm trying to push the final styling for later because I actually want to use Chromabox to generate the colors 😁

**Pull request merged**